Source/Core/Common/WindowsRegistry.h

```diff
@@ -2,6 +2,9 @@
 
 #include <Windows.h>
 
+#include <Common/CommonTypes.h>
+#include <string>
```

Comment:
```
#include <string>
#include <Windows.h>

#include "Common/CommonTypes.h"
```

Reply:
Hmm... Lint doesn't like that order

Reply:
Ah, it must be because of the capitalization. I figured W comes after s but it doesn't.

Reply:
At the very least it should be `"Common/CommonTypes.h"` instead of `<Common/CommonTypes.h>`, I'd think.